scheduled-jobs/build/rhel7next/Jenkinsfile

```diff
@@ -0,0 +1,39 @@
+properties([[
+  $class: 'ParametersDefinitionProperty',
+  parameterDefinitions: [
+    [
+      $class: 'hudson.model.BooleanParameterDefinition',
+      name: 'MOCK',
+      description: 'Mock run to pickup new Jenkins parameters?',
+      defaultValue: false,
+    ],
+  ],
+]])
+
+node('openshift-build-1') {
+  try {
+    dir('aos-cd-jobs') {
+      def buildlib = null
+      stage('clone') {
+        checkout scm
+        buildlib = load('pipeline-scripts/buildlib.groovy')
+      }
+      stage('sync') {
+        sshagent(['openshift-bot']) {
```

Comment:
Is this key necessary to ssh to `rcm-guest`?

Reply:
@bbguimaraes No, the key for rcm-guest just sits on the buildvm filesystem.